workflowRDF/fairworkflow.py

```diff
@@ -1,21 +1,108 @@
+import os
+import tempfile
 import rdflib
 from rdflib.namespace import RDF, RDFS, DC, XSD, OWL 
 import inspect
 from datetime import datetime
 
-PPLAN = rdflib.Namespace("http://purl.org/net/p-plan/")
-PLEX = rdflib.Namespace("https://plex.org/")
-EDAM = rdflib.Namespace("http://edamontology.org/")
-PROV = rdflib.Namespace("http://www.w3.org/ns/prov/")
+# Some standard ontologies used for nanopubs and describing workflows.
+NP = rdflib.Namespace("http://www.nanopub.org/nschema#")
+PPLAN = rdflib.Namespace("http://purl.org/net/p-plan#")
+PROV = rdflib.Namespace("http://www.w3.org/ns/prov#")
 DUL = rdflib.Namespace("http://ontologydesignpatterns.org/wiki/Ontology:DOLCE+DnS_Ultralite/")
 BPMN = rdflib.Namespace("https://www.omg.org/spec/BPMN/")
 PWO = rdflib.Namespace("http://purl.org/spar/pwo/")
-RDFG = rdflib.Namespace("http://www.w3.org/2004/03/trix/rdfg-1/")
-NP = rdflib.Namespace("http://www.nanopub.org/nschema#")
+
+
+class Nanopub:
```

Comment:
Since it is all static methods I would put it in a separate module and get rid of the class. But I could also do that if I merge the code with the core package.